packages/app/src/systems/Asset/components/AssetInput/AmountInputLoader.tsx

```diff
@@ -0,0 +1,23 @@
+import { darkColors } from '@fuel-ui/css';
+import { CardList } from '@fuel-ui/react';
+import type { IContentLoaderProps } from 'react-content-loader';
+import ContentLoader from 'react-content-loader';
+
+export const AmountInputLoader = (props: IContentLoaderProps) => (
```

Comment:
this loader is looking equal to `AssetItem` loader. which is wrong.

it should look similar to `AmountInput` component. look the video with both `AmountInput` and `AssetItem` examples:


https://user-images.githubusercontent.com/8636507/200440753-9cbfbc8a-b755-4104-a539-4ff3f1680f7d.mov

Reply:
good catch!  I initially just took the loader from AssetItem and forgot to change it